site/docs/get-started/quickstart.md

```diff
@@ -1,61 +1,93 @@
 # Quick Start Guide
 
-This guide lets you quickly evaluate Mesh for Data. 
+Follow this guide to install Mesh for Data using default parameters that are suitable for experimentation on a single cluster.
+
+<!-- For a full installation refer to the [full installation guide](./setup/install) instead. -->
 
 ## Before you begin
 
 Ensure that you have the following:
 
-- `git`
-- `make`
-- `kubectl` version 1.16 and above
-- `helm` version 3.3 and above
+- [Helm](https://helm.sh/) 3.3 or newer must be installed and configured on your machine.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as [Kind](http://kind.sigs.k8s.io/) as a cluster administrator.
 
-## About this guide
+## Install cert-manager
 
-By the end of this guide you will have the following installed to your Kubernetes cluster:
+Mesh for Data requires [cert-manager](https://cert-manager.io) to be installed to your cluster. 
+Many clusters already include cert-manager.
+Check if `cert-manager` namespace exists in your cluster and only run the following if it does **not** exist:
 
-- The control plane of Mesh for Data
-- Hashicorp Vault and connector as the credentials manager 
-- ODPi Egeria lab and connectors as the data catalog
-- Open Policy Agent (OPA) and connector as the policy manager.
-- Arrow-Flight data access module for reading Parquet and CSV datasets
+```bash
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 120s
+``` 
 
-## Prepare for installing Mesh for Data
+## Install Hashicorp Vault and plugins
 
-1.  Obtain a local copy of Mesh for Data repository
-    ```bash
-    git clone https://github.com/ibm/the-mesh-for-data.git
-    ```
-1.  Change to the root directory of the repository
-    ```bash
-    cd the-mesh-for-data
-    ```
+[Hashicorp Vault](https://www.vaultproject.io/) and a [secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) plugin are used by Mesh for Data for credential management.
 
-## Install Mesh for Data
+Run the following to install vault and the plugin in development mode:
 
-1. Set the current namespace to `m4d-system`
+=== "Kubernetes" 
 
     ```bash
-    kubectl config set-context --current --namespace=m4d-system
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/values.yaml \
+        --wait --timeout 120s
```

Comment:
Maybe we can give some instructions on how to handle timeout errors ? I am getting the error given below (I retried 2 times but still getting this error)

<img width="1255" alt="image" src="https://user-images.githubusercontent.com/59818576/115145162-d2982d80-a06d-11eb-997d-b12431da0ad7.png">

Reply:
@rohithdv i believe that Helm by default check the "default" namespace.
do helm list -A to see installed charts and use helm uninstall -n vault vault to set the target namespace.

Re the error, did you try to increase the timeout?